drivers/SmartThings/zigbee-contact/src/test/test_smartsense_multi.lua

```diff
@@ -290,6 +290,48 @@ test.register_coroutine_test(
     end
 )
 
+test.register_coroutine_test(
+  "Report from cluster 0xFC03, command 0x09 should be handled as: Temperature (-25.5 C), Acceleration - active, Contact - closed, battery(97%)",
+  function()
+    test.socket.zigbee:__queue_receive({
+      mock_device.id,
+      build_multi_status_report_message(mock_device, "\x04\x01\x00\x48")
+    })
+    test.socket.capability:__set_channel_ordering("relaxed")
+    test.socket.capability:__expect_send(mock_device:generate_test_message("main", capabilities.temperatureMeasurement.temperature({ value = -25.5, unit = "C" })))
+    test.socket.capability:__expect_send(mock_device:generate_test_message("main", capabilities.accelerationSensor.acceleration.active()))
+    test.socket.capability:__expect_send(mock_device:generate_test_message("main", capabilities.contactSensor.contact.closed()))
+    test.socket.capability:__expect_send(mock_device:generate_test_message("main", capabilities.battery.battery(97)))
+  end)
+
+test.register_coroutine_test(
```

Comment:
I suggest testing different temperatures in each of these new tests. The handling of acceleration, contact and battery is tested by other tests so it'd be more valuable here is to test the edge cases for temp handling.

Reply:
added more tests with different negative values (31500af800ee206a8382804c95f8d55d66a39d53)